src/scripts/extensions/injectHelper.ts

```diff
@@ -1,12 +1,19 @@
 import {Localization} from "../localization/localization";
+import { WebExtension } from "./webExtensionBase/webExtension";
 
 export class InjectHelper {
 	private static uninjectableUrlRegexes: RegExp[] = [
 		/^about:/
 	];
 
 	public static alertUserOfUnclippablePage() {
-		alert(Localization.getLocalizedString("WebClipper.Error.CannotClipPage"));
+		WebExtension.browser.notifications.create({
+			type: 'basic',
+			iconUrl: '/icons/icon-48.png',
+			title: 'Cannot Clip Page',
```

Comment:
Shall we localize the string for title as well?

Reply:
Done